.github/workflows/install_test_setup_py.yml

```diff
@@ -23,3 +23,13 @@ jobs:
       run: python setup.py install
     - name: Test through setup.py
       run: python setup.py test
+    - name: Install coverage
+      run: pip install coverage
+    - name: Create coverage report
+      run: coverage run --omit '/usr/*' ./setup.py test && coverage xml
```

Comment:
This is gonna run tests twice again, if I understand correctly. Maybe we can skip the test run in L 24 / 25?

Also, this will run coverage reports for all versions of python as defined in the matrix (so about 6 times). Maybe having a separate action to run coverage would be simpler in this case.

Reply:
Correct; coverage migrated to its own GitHub action.